src/frontend/components/StudentLiveAdvertising/StudentLiveRegistration/index.tsx

```diff
@@ -50,15 +46,15 @@ export const StudentLiveRegistration = () => {
   useLayoutEffect(() => {
```

Comment:
you can use a useEffect here now, there is no longer any reason to use a useLayoutEffect

Reply:
:+1: